libs/blocks/caas-config/caas-config.js

```diff
@@ -408,6 +412,9 @@ const TagsPanel = ({ tagsData }) => {
     >
       <${TagSelect} id="orTags" options=${allTags} label="Tags"
     /><//>
+    <label>Advanced Tag Configurations</label>
+    <${Input} label="Use a secondary source for some content types" prop="showSecondarySource" type="checkbox" />
```

Comment:
Minor (subjective) edit: 
Change label to "Use a secondary source"

Reply:
I think since you can use multiple sources in the normal configurator - the clarification that this is specifically for when you want to restrict content types for this source is necessary